src/libs/BootSplash/index.native.js

```diff
@@ -1,3 +1,15 @@
 import RNBootSplash from 'react-native-bootsplash';
+import Log from '../Log';
 
-export default RNBootSplash;
+export default {
+    // react-native-bootsplash 3.x could reject if not init (ex: when opening app via iOS >= 15 notifications)
+    // As the splash screen is not visible on these cases, we chose to ignore these errors.
+    hide: config => RNBootSplash.hide(config).catch((error) => {
+        Log.alert('[BootSplash] hiding failed', {message: error.message, error}, false);
+    }),
+    show: config => RNBootSplash.show(config).catch((error) => {
+        Log.alert('[BootSplash] showing failed', {message: error.message, error}, false);
+    }),
```

Comment:
If we're no longer using `show`, and won't be able to use it in the next major version we might as well remove it

This would leave only a single `Log` call here, so we might leave it in the original place where Log is imported anyway

Also this `export` configuration does not match the style guide. I'm think it's specified somewhere, but we first define `functions` and then export them